packages/server/lib/controllers/sync.controller.ts

```diff
@@ -761,4 +764,34 @@ class SyncController {
     }
 }
 
+function normalizedSyncParams(syncs: any): Result<{ syncName: string; syncVariant: string }[]> {
+    if (!syncs) {
+        return Err('Missing sync names');
+    }
+    if (!Array.isArray(syncs)) {
+        return Err('syncs must be an array');
+    }
+
+    const syncIdentifiers = syncs.map((sync) => {
+        if (typeof sync === 'string') {
+            if (sync.includes('::')) {
+                const [name, variant] = sync.split('::');
+                return { syncName: name, syncVariant: variant };
+            }
```

Comment:
agreed with @bastienbeurier to support `sync_name::sync_variant` format for the GET `/sync/status` endpoint where syncs are passed as a query param